src/hooks/useWallets.ts

```diff
@@ -19,50 +15,29 @@ const walletSelector = createSelector(
   }),
   ({ isWalletLoading, selectedWallet, walletNames, wallets }) => ({
     isWalletLoading,
-    latestBackup: findLatestBackUp(wallets),
     selectedWallet,
     walletNames,
     wallets,
   })
 );
 
 export default function useWallets() {
-  const initializeWallet = useInitializeWallet();
-  const dispatch = useDispatch();
-  const { isWalletLoading, latestBackup, selectedWallet, walletNames, wallets } = useSelector(walletSelector);
+  const { isWalletLoading, selectedWallet, walletNames, wallets } = useSelector(walletSelector);
 
-  const isDamaged = useMemo(() => {
-    const bool = selectedWallet?.damaged;
-    if (bool) {
-      logger.sentry('Wallet is damaged. Check values below:');
-      logger.sentry('selectedWallet: ', selectedWallet);
-      logger.sentry('wallets: ', wallets);
-    }
-    return bool;
-  }, [selectedWallet, wallets]);
-
-  const switchToWalletWithAddress = async (address: string): Promise<string | null> => {
-    const walletKey = Object.keys(wallets!).find(key => {
-      // Addresses
-      return wallets![key].addresses.find((account: RainbowAccount) => account.address.toLowerCase() === address.toLowerCase());
-    });
+  const isDamaged = selectedWallet?.damaged;
 
-    if (!walletKey) return null;
-    const p1 = dispatch(walletsSetSelected(wallets![walletKey]));
-    const p2 = dispatch(addressSetSelected(toChecksumAddress(address)!));
-    await Promise.all([p1, p2]);
-    // @ts-expect-error ts-migrate(2554) FIXME: Expected 8-9 arguments, but got 7.
-    return initializeWallet(null, null, null, false, false, null, true);
-  };
+  useEffect(() => {
```

Comment:
should we remove this from here as this would log per instance of the `useWallets` hook? we might need to do a separate follow up to find a better place for it to be logged

Reply:
I was thinking this too. Here's a ticket https://linear.app/rainbow/issue/APP-1778/add-logging-for-damaged-wallets